config/boards/README.md

```diff
@@ -27,6 +31,13 @@
 | PACKAGE_LIST_BOARD_REMOVE | space delimited packages to be removed |
 | PACKAGE_LIST_DESKTOP_BOARD | space delimited packages to be installed on this boards desktop build |
 | PACKAGE_LIST_DESKTOP_BOARD_REMOVE | space delimited packages to be removed |
+| ASOUND_STATE | ALSA sound settings |
+| ATFBRANCH | custom ARM Trusted Firmware branch |
+| BOOT_FDT_FILE | u-boot "fdtfile" environment value  |
```

Comment:
"Forcing loading specific device tree configuration - if its different than the one defined by u-boot"